src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -173,6 +191,14 @@
                         {@line}
                     @end
                 """
+                if '{@apiMethod.name}' not in self._stubs:
+                    self._stubs['{@apiMethod.name}'] = google.api_core.gapic_v1.method.wrap_method(
```

Comment:
Oh, I see the problem with naming this `_stubs` -- this is slightly different than what gRPC calls a "stub" (it is using the term for the overall class).

Well, grumble.

Okay, so sorry, can this change _again_ to `_inner_api_calls`?

Reply:
Changed to _inner_api_calls.